docs/troubleshooting/kubernetes.rst

```diff
@@ -1,13 +1,223 @@
 Troubleshoot Your Kubernetes Deployment
 =======================================
 
-BIG-IP Controller
------------------
+.. toctree::
+   :maxdepth: 1
+
+How to get help
+---------------
+
+If the issue you're experiencing isn't covered here, try one of the following options:
+
+- `Contact F5 Support`_ (valid support contract required).
+- `Report a bug <https://github.com/F5Networks/k8s-bigip-ctlr/issues>`_ in the k8s-bigip-ctlr GitHub repo.
+- `Ask a question <https://f5cloudsolutions.slack.com>`_ in the #cc-kubernetes channel in the F5 Cloud Solutions Slack team.
```

Comment:
We should include the https://f5cloudsolutions.herokuapp.com/ URL. Also direct them that this is where to get an invite.